BUILD

```diff
@@ -40,14 +46,12 @@ genrule(
 genrule(
     name = "android_dist",
     srcs = [
-        "//library/kotlin/src/io/envoyproxy/envoymobile:android_aar",
+        "//library/kotlin/src/io/envoyproxy/envoymobile:android_aar_only_aar",
```

Comment:
I think this name is a little funny. I would presume that an `android_aar` target would _only_ build the aar. i.e, I would expect the target that builds more to be the one that has a suffix: `envoy_aar_with_javadocs`?

Reply:
Hmm...How about `android_artifacts` since it generates other things too